pkg/db/metal/network.go

```diff
@@ -11,8 +16,10 @@ type (
 		Prefixes                   Prefixes          `rethinkdb:"prefixes"`
 		DestinationPrefixes        Prefixes          `rethinkdb:"destinationprefixes"`
 		DefaultChildPrefixLength   ChildPrefixLength `rethinkdb:"defaultchildprefixlength" description:"if privatesuper, this defines the bitlen of child prefixes per addressfamily if not nil"`
+		MinChildPrefixLength       ChildPrefixLength `rethinkdb:"minchildprefixlength" description:"if privatesuper, this defines the minimum bitlen of child prefixes per addressfamily if not nil"`
 		PartitionID                string            `rethinkdb:"partitionid"`
 		ProjectID                  string            `rethinkdb:"projectid"`
+		Namespace                  *string           `rethinkdb:"namespace" description:"if this is a namespaced private network, the namespace is stored here, otherwise nil"`
 		ParentNetworkID            string            `rethinkdb:"parentnetworkid"`
 		Vrf                        uint              `rethinkdb:"vrf"`
 		PrivateSuper               bool              `rethinkdb:"privatesuper"`
```

Comment:
Just a suggestion but this could be done without breaking change (just keep old field in the database):

```suggestion
		Super               bool              `rethinkdb:"privatesuper"`
```

Reply:
Maybe not worth the effort, i would like to deprecate this field then